### DISCUSSION TITLE
how to save generated rust code?

### DISCUSSION THREAD

**Discussion by zzzz465** [hyperium/tonic#1092] [Q&A] [answered]
I'm using a large proto file and experiencing IDE lag after adding tonic.
I assume that generating rust codes on-the-fly causes this? so I'm trying to put generated code as static.
(like go-protobuf)

+ IDE lagging by 10~15 seconds.

**Comment by davidpdrsn** [comment, marked as answer]
You can set https://docs.rs/tonic-build/0.8.2/tonic_build/struct.Builder.html#method.out_dir to somewhere else, probably in `src`.
